### PR TITLE
Assignment 4 to Dev

### PR DESCRIPTION
**Pull request by anthonynmh**

### CODE REVIEW

**Review by JrmCkh — APPROVED** [review]
LGTM

File: Question/helper/callsToAuth.js
Comment (venuslimm) [inline Question/helper/callsToAuth.js:1]: Is it possible to rename helper to helpers folder to standardise?

File: Question/QuestionRoute.js
Comment (venuslimm) [inline Question/QuestionRoute.js:1]: Is it possible to rename QuestionRoute to QuestionRoutes to standardise?

**Review by venuslimm — COMMENTED** [review]
Left some comments for standardising

**Pull request merged**